src/lib/components/comparisonTable/components/AccordionItem/AccordionItem.module.scss

```diff
@@ -0,0 +1,44 @@
+@import '../../../../scss/public/colors';
+
+.container {
+  width: var(--growContent, min-content);
+  margin: 0 auto;
+}
+
+.container + .container {
```

Comment:
Is this a typo?

Reply:
No, it's meant to add distance between containers ("if a container follows a container")

Reply:
Which means the very first container won't have that margin.